docs/docs/recipes/integrate-logto/vanilla-js.mdx

```diff
@@ -0,0 +1,108 @@
+---
+sidebar_label: Vanilla JS
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import AppNote from './components/AppNote';
+import SignInNote from './components/SignInNote';
+import SignInFlowSummary from './fragments/_web-sign-in-flow-summary.mdx';
+import SignOutNote from './fragments/_web-sign-out-note.md';
+
+# Vanilla JS: Integrate `@logto/browser`
+
+:::tip
+The browser SDK is framework-agnostic, meaning you can use it in any frontend framework by writing a wrapper to fit it. That's what `@logto/react` and `@logto/vue` did.
+:::
+
+<AppNote type="Single Page" />
+
+## Add Logto SDK as a dependency
+
+<Tabs>
+  <TabItem value="npm" label="npm">
+
+```bash
+npm i @logto/browser
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @logto/browser
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```bash
+pnpm add @logto/browser
+```
+
+  </TabItem>
+</Tabs>
+
+## Init LogtoClient
+
+Import and init `LogtoClient` by passing config:
+
+```tsx
+import LogtoClient from '@logto/browser';
+
+const logtoClient = new LogtoClient({
+  endpoint: '<your-logto-endpoint>', // E.g. http://localhost:3001
+  appId: '<your-application-id>',
+});
```

Comment:
Should we remind the user that this `logtoClient` should be used as a singleton instance, and any forms of re-initialization should be avoided

Reply:
i think it's okay since `new` already implied this